packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -38,6 +40,24 @@ interface Props {
   dataCy?: string
 }
 
+const ColorBadge = styled('div')<{groupColor: string}>(({groupColor}) => ({
+  backgroundColor: groupColor,
+  height: 32,
+  width: 32
+}))
+
+const BadgeWrapper = styled('div')({
+  borderTopLeftRadius: 30,
+  borderBottomRightRadius: 100,
+  height: 16,
+  width: 16,
+  left: 0,
+  top: 0,
+  overflow: 'hidden',
+  position: 'relative',
```

Comment:
design question: why no position: 'absolute' to keep the height the same?

Reply:
Good point! I think this was actually a mistake, changed to `absolute` now.